src/custom/constants/routes.ts

```diff
@@ -0,0 +1,24 @@
+// ENUM with routes
+export enum Routes {
```

Comment:
@fairlighteth this I like 👍
we should make sure the all our links start to use this enums. Maybe we can even create a Link which only accepts this enum as the path parameter